test/unit/utils/get.test.js

```diff
@@ -46,4 +46,10 @@ describe('Utils.get', function () {
     expect(fallback).toBe(true)
   })
 
+  it('returns undefined if the fallback is undefined', function () {
+    let fallback = get(null, null)
+
+    expect(fallback).toBe(undefined)
+  })
```

Comment:
What do you think about re-writing this as `fallback = get({}, 'key')`. Or do you think we should cover both cases?